game-core/src/test/java/games/strategy/engine/ClientContextIntegrationTest.java

```diff
@@ -4,10 +4,12 @@
 import static org.hamcrest.Matchers.notNullValue;
 import static org.hamcrest.core.Is.is;
 
+import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
 import games.strategy.triplea.settings.AbstractClientSettingTestCase;
 
+@Tag("integration")
```

Comment:
Would it make sense to extract the literal `"integration"` to a constant to avoid the potential for fat-fingering it?  For example:

```java
package games.strategy.test;

public final class Tags {
  public static final String INTEGRATION = "integration";

  private Tags() {}
}
```

so that you can write:

```java
@Tag(Tags.INTEGRATION)
public class ClientContextIntegrationTest extends AbstractClientSettingTestCase {
```

and have the compiler check it?

Reply:
There is a better scheme for this...
Let me do that real quick.